### PR TITLE
chore: remove deprecated usage of less expression

### LINKED ISSUE #2884: [FEATURE]: remove less file ~ import
**如果你的需求与问题相关，请在下面描述一下（Is your feature request related to a problem? Please describe.）**

根据 webpack 的建议，使用 ~ import 库中的 less 已经被废弃，不再推荐使用
https://webpack.js.org/loaders/less-loader/#webpack-resolver
image

**描述你预期的功能表现（Describe the solution you'd like）**

希望代码中使用正确的引用方式代替

**描述你考虑过的替代方案（Describe alternatives you've considered）**

**补充信息（Additional context）**

使用正确的引用方式也有助于切换到其他的打包器如: vite 等

### PR DESCRIPTION
**Pull request by erha19**
### Types

- [x] 🧹 Chores

### <samp>🤖 Generated by Copilot at 58eb995</samp>

*  Changed import paths of `@opensumi/ide-components` and `@opensumi/ide-core-browser` from using `~` to using relative paths in various `.less` files to avoid potential conflicts with webpack alias or other loaders ([link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-4c0119657bc88abfae15a129671e5d525d83d43e011da50d917df1e1ccee9cadL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-e0c30a50911943639dd78782678bc9140f085740da214eeb804b1817c310c306L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-4597d7ec10e6ce2e45ab08755a8583d676de6b76c163d000aa4b100f82e470f8L8-R8), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-ed9322c6c5533fc2b9c68b6d7bbc37cd25d85ffb6f0086a1fe4595ca6b61641bL1-R4), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-620db825c2ddc84072c0fc0983496c29d141287910f61ba3004525619281bd3dL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-78716d418280bb462c90bbb02a3055c27022ce72c628a02dcefd70d03f7bbc58L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-de3b9f29f1b757cc7f3b63a9728ae7b99dfb3762a06932f572b4f238bf82cbcdL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-d19dafb38002b9cf10c585b975ee221bbc4ceac775f2635e45cdff8b628b0cecL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-a92d3c79f3be1ec7533bcb9d276d6cacbd8a6a9df61391c4cfd7fcf805025ce1L1-R2), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-47e5c21acc3df52792bb2d9da11e12675e03e24c4e76d0824f689c6d92f3d9b7L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-1fb3b357c52030563876fcb950ab18a4dcb163724362271de47e29821acd62b7L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-e0b9cac329c9ee96aff82882e1249772e352a20efbf5a5bd0ea211c64c4fdc9aL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-aee48a3591970fb895cdbe98779e3fbf1eee08c59b4608170f88b13ec7aa6926L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-420086e5466afed9765091eb57763ecf0dcf7d5c416bf8dd8340ca9dd2225f8fL1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-6c53f27ebf14014bc474df3d6fbed11c6e8e5b126608ce5238c9d901ec5dd4e4L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-72cacce28fa24b09c05928131c620bf9fb3f67b0a1268ddf3737c0f425905280L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-34a101b784633edf5acc43308559ca4420f19934d1e51b119329bbc5d7da5499L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-d3a47c29ae29a77aae8603a48d4aa4073e6543bf32b5178faa899c5333085934L1-R1), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-a6ce8684ec41df8e27177c2d9b465ade4e3159b9ee2e462327e2933e62930a6dL1-R1))
* Added blank lines to `packages/components/README.md` to improve readability and formatting in the `Usage` and `Icon/createFromIconfontCN` sections ([link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-4b24799689b090701f0bc36ea0a508a962d448d1feaec58750b8fcd61e150c21R6), [link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-4b24799689b090701f0bc36ea0a508a962d448d1feaec58750b8fcd61e150c21R36))
* Adjusted indentation of the `type IconFontMap` declaration in `packages/components/README.md` to align with the following code block in the `Icon/createFromIconfontCN` section ([link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-4b24799689b090701f0bc36ea0a508a962d448d1feaec58750b8fcd61e150c21L12-R13))
* Added a space between `本地` and `IDE` in the comment of `packages/core-browser/src/style/icon.less` for consistency ([link](https://github.com/opensumi/core/pull/2906/files?diff=unified&w=0#diff-ed9322c6c5533fc2b9c68b6d7bbc37cd25d85ffb6f0086a1fe4595ca6b61641bL1-R4))

close #2884

### <samp>🤖 Generated by Copilot at 58eb995</samp>

This pull request fixes some import paths and documentation issues in various packages. It changes the import paths of `@opensumi/ide-components` and `@opensumi/ide-core-browser` from using `~` to using relative paths to avoid potential conflicts with webpack alias or other loaders. It also fixes some formatting and import issues in `packages/components/README.md` and aligns the `type IconFontMap` declaration with the code style of the rest of the file.

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -53,13 +51,14 @@ renderApp({
...
-* components 包的定位是直接能对外使用，不限定于在 ide-framework 运行环境的组件
-* 因此需要注意的是，放到这里的组件应该是纯组件
-  * 不包含对 ide-framework runtime 依赖
-  * 不包含对 ide-framework 其他包的依赖
-  * 自己的依赖自己管理
-* 脱离 ide-fw 去单独使用 `@opensumi/ide-components` 时，应手动 import 字体文件
+
+- components 包的定位是直接能对外使用，不限定于在 ide-framework 运行环境的组件
+- 因此需要注意的是，放到这里的组件应该是纯组件
+  - 不包含对 ide-framework runtime 依赖
```
Comment (bytemain) [inline packages/components/README.md:57]: 改成 opensumi 吧

File: packages/testing/src/browser/outputPeek/test-peek-widget.less [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// @import '~@opensumi/ide-monaco-enhance/lib/browser/styles.module.less';
+// @import '@opensumi/ide-monaco-enhance/lib/browser/styles.module.less';
```
Comment (bytemain) [inline packages/testing/src/browser/outputPeek/test-peek-widget.less:1]: 就删了吧

Comment (erha19) [reply packages/testing/src/browser/outputPeek/test-peek-widget.less]: done

File: packages/components/README.md [hunk 3]
```diff
@@ -53,13 +51,14 @@ renderApp({
...
-  * 不包含对 ide-framework 其他包的依赖
-  * 自己的依赖自己管理
-* 脱离 ide-fw 去单独使用 `@opensumi/ide-components` 时，应手动 import 字体文件
+
+- components 包的定位是直接能对外使用，不限定于在 OpenSumi 运行环境的组件
+- 因此需要注意的是，放到这里的组件应该是纯组件
+  - 不包含对 OpenSumi runtime 依赖
+  - 不包含对 OpenSumi 其他包的依赖
+  - 自己的依赖自己管理
+- 脱离 ide-fw 去单独使用 `@opensumi/ide-components` 时，应手动 import 字体文件
```
Comment (bytemain) [inline packages/components/README.md:60]: opensumi

**Review by bytemain — APPROVED** [review]

**Pull request merged**